app/stac_api/management/commands/perftest.py

```diff
@@ -0,0 +1,118 @@
+        range(2570000, 2746000, 1000)
+        range(1130000, 1252000, 1000)
+
+        if options['clean']:
+            Collection.objects.filter(name__startswith='perftest').delete()
+        elif options['populate']:
+
+            for collection_id in [f'perftest-collection-{x}' for x in range(30)]:
+                collection, _ = Collection.objects.get_or_create(
+                    name=collection_id,
+                    defaults={
+                        'description': 'This is a description',
+                        'license': 'test',
+                        'summaries': {
+                            "eo:gsd": [], "geoadmin:variant": [], "proj:epsg": []
+                        },
+                        'title': 'Test title'
+                    }
+                )
+                for _xmin in range(XMIN, XMAX, 1000):
+                    for _ymin in range(YMIN, YMAX, 1000):
+                        geo = Polygon.from_bbox((_xmin, _ymin, _xmin + 1000, _ymin + 1000))
+                        geo.srid = 2056
+                        geo.transform(4326, clone=False)
+                        item_id = f'perftest-item-{_xmin}-{_ymin}'
+
+                        item, _ = Item.objects.get_or_create(
+                            collection=collection,
+                            name=item_id,
+                            defaults={
+                                'properties_datetime': random_datetime(MIN_DATETIME, MAX_DATETIME),
+                                'properties_title': "My Title",
+                                'geometry': geo
+                            }
+                        )
+
+                        for asset_id in [f'perftest-asset-{x}' for x in range(3)]:
+                            asset, _ = Asset.objects.get_or_create(
+                                item=item,
+                                name=asset_id,
+                                defaults={
+                                    'title': 'my-title',
+                                    'checksum_multihash': "01205c3fd6978a7d0b051efaa4263a09",
+                                    'description': "this an asset",
+                                    'eo_gsd': random.choice([2, 2.5, 5, 10]),
+                                    'geoadmin_lang': random.choice(['de', 'fr', 'it', 'rm', 'en', '']),
+                                    'geoadmin_variant': random.choice(['var1', 'var2', 'var3']),
+                                    'proj_epsg': random.choice([2056, 4326, 21781]),
+                                    'media_type': random.choice([typ[0] for typ in MEDIA_TYPES]),
+                                    'file': SimpleUploadedFile(
+                                        f'{item.collection.name}/{item.name}/{asset_id}',
+                                        b"dummy content"
+                                    )
+                                }
+                            )
+                            asset.save()
```

Comment:
The `save()` here is not needed, `get_or_create()` already save the asset.

Reply:
removed